### PR TITLE
Added Mailing Script

### LINKED ISSUE #252: Implementing mailing script in master branch
We need to implement mailing script in master branch. It is already there in generator branch and it needs to be export to the master branch.
Mailing script is used to send mail with the ISO link to the user who requested for the ISO through Meilix Generator webapp

### PR DESCRIPTION
**Pull request by meets2tarun**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
Fixes #252
### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -40,8 +40,15 @@ deploy:
...
+      secure: "Bzwk661qPFSBm14n3cOpaTR8ELei7Fw9lvKOGHlKyU/VpR3YB2qqF8eIsEmgzB4vpi6v6tpcZv/dpseSFn9NG4un75s5cZ1//yFp0KGaRGDcMM2W4+j3RTmHPrvp46X2z3lXGvwY+kfDTmv5px6+SuDpVDMKUgnk+wp7JvGG3FNItUK9WfF6MTwGCp4A/KIxLtKCCWxQhL3g8Gp8vNWBTXU6ZLrF/PtQyz8yZv3N01b86TfpM7DHYnZD4cCxam4YPyCzlOqE30kKYz9RkXSJCFepcBAni8kGkuGs5j34qf4IEck7E9/qWF2OOTtCa7mwxCnpIPl5uVhrMi+6XRMuRIr8P3+fuTGCrAhEeY1zdVX1wqrw0AgQNCfz288GLVZdbc4iV6MiZAPIwc4nNuP4WJBMCC1WUEsNK6a5GsVPNG1KP5Q2mVIQE1KJj9/Wv0LWglKnG2DA1hQd5Xu/hIUiMkzO0+8aYM0eM54AkHKCR9Drv90kTwcGhFYc7fQWVa4YXXVDzdfaOGn0EUR6ePN7Vg0KRKkDbHQ2yg4rZJsYg1tkRxd1XgGb1HCaeXZA7KVcwXIQXf4bkp/lRDholc81yYViCG5oR5wm1xdFIJVj7dP8Z55aPh081KbHVhUzXoTvEWnS9fOot0fG7DgPBDcVLpeMdmKLjid4no7lbyPB2qw="
```
Comment (niccokunzmann) [inline .travis.yml]: Why did you change this? Maybe this is not required and we can remove it because no one looks at it.

Comment (meets2tarun) [reply .travis.yml]: @niccokunzmann 
Yes, I'll change it.
